src/js/background.js

```diff
@@ -140,9 +141,29 @@ Badger.prototype = {
 
   // Methods
 
+  // load seed dataset with pre-trained action and snitch maps
+  loadSeedData: function() {
+    let self = this;
+    utils.xhrRequest(constants.SEED_DATA_LOCAL_URL, function(err, response) {
+      if (!err) {
+        var seed = JSON.parse(response);
+        self.storage.getBadgerStorageObject("action_map").merge(seed.action_map);
```

Comment:
Should we use `badger.mergeUserData` here instead?

Reply:
Maybe -- there will be the extra "version" key in the imported json, can the function handle that?

Reply:
(yes, it can, and yes, we should.)